dotCMS/src/main/webapp/WEB-INF/jsp/es-search/not_licensed.jsp

```diff
@@ -1,20 +1,33 @@
 <%@page import="com.liferay.portal.language.LanguageUtil"%>
 <style>
-	#dotAjaxMainDiv {overflow: visible;}
-	.wrapperCluster{background:url(/html/images/skin/es-search-promo.png) no-repeat 50% 0;height:757px;margin:0 auto;border:1px silver solid;padding:0px;overflow:hidden;}
-	.content{position:fixed;left:50%;top:50%;margin:-200px 0 0 -300px;width:600px;background:#333;opacity:.75;color:#fff;padding:20px 20px 35px 20px;-moz-border-radius: 15px;-webkit-border-radius: 15px;-moz-box-shadow:0px 0px 15px #666;-webkit-box-shadow:0px 0px 15px #666;}
-	.content h2{font-size:200%;}
-	.content p{margin:0;}
-	.content ul{margin:5px 0 25px 15px;padding:0 0 0 10px;list-style-position:outside; list-style:decimal;}
-	.content li{list-style-position:outside; list-style:disc;}
-	.content a{color:#fff;}
+    #dotAjaxMainHangerDiv, #dotAjaxMainDiv { height: 100%; }
+    .portlet-wrapper { display: flex; height: 100%; justify-content: center; }
+    .unlicense-content { align-self: center; text-align: center; }
+    .unlicense-content i, .unlicense-content h4 { color: #b3b1b8 }
+    .unlicense-content h4 { font-size:24px; font-weight:400; line-height:36px; }
+    .unlicense-content p { font-size:14px; font-weight:400; line-height:21px; margin-block-end:14px; margin-block-start:14px; }
+    .unlicense-content ul { display: inline-block; margin-block-end: 14px; margin-block-start: 14px;}
+    .unlicense-content ul li { line-height: 21px; list-style-type:disc; text-align: left; }
+    .unlicense-content ul li a { color: var(--color-main); font-size:14px; }
+    .request-license-button { background: var(--color-main); border: solid 1px transparent; border-radius: 2px; color: #fff; display: inline-block; line-height: 36px; padding: 0 24px; text-decoration: none; text-transform: uppercase; }
+    .request-license-button:hover { background: var(--color-main_mod); border: solid 1px transparent; }
 </style>
-
-<div class="wrapperCluster">
-	<div class="content">
-		<h2><%=LanguageUtil.get(pageContext, "com.dotcms.repackage.javax.portlet.title.es-search")%></h2>
-		<p><%= LanguageUtil.get(pageContext, "ES-QUERY-NOT-LICENSED") %></p>
-	</div>
+<div class="portlet-wrapper">
+    <div class="unlicense-content">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5587482)